Sources/BraintreePayPalNativeCheckout/BTPayPalNativeOrderCreationClient.swift

```diff
@@ -28,16 +28,19 @@ class BTPayPalNativeOrderCreationClient {
     ) {
         apiClient.fetchOrReturnRemoteConfiguration { configuration, error in
             guard let config = configuration, error == nil else {
+                self.apiClient.sendAnalyticsEvent("ios.paypal-native-checkout.create-order.fetch-configuration.failure")
```

Comment:
I don't think we need any config fetch analytics. None of our other payment method features have them, since the actual config fetch call already has analytics baked into it.